castai/resource_workload_scaling_policy.go

```diff
@@ -125,6 +125,21 @@ func resourceWorkloadScalingPolicy() *schema.Resource {
 					},
 				},
 			},
+			"anti_affinity": {
```

Comment:
Name is a bit misleading IMO. What does it mean to set anti-affinity to true or false? I think our API has a better name `consider_anti_affinity` maybe we can keep it in line with API naming?